crypto/ec/curve448/field.h

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2014 Cryptography Research, Inc.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Originally written by Mike Hamburg
+ */
+
+#ifndef __GF_H__
+# define __GF_H__
+
+# include "internal/constant_time_locl.h"
+# include <string.h>
+# include <assert.h>
+# include "word.h"
+
+# define NLIMBS (64/sizeof(word_t))
+# define X_SER_BYTES 56
+# define SER_BYTES 56
+
+# if defined(__GNUC__) || defined(__clang__)
+#  define INLINE_UNUSED __inline__ __attribute__((unused,always_inline))
+#  define RESTRICT __restrict__
+#  define ALIGNED __attribute__((aligned(32)))
```

Comment:
this requests 32 _byte_ alignment. The alignment is not measured in bits here.
Was that the intention?
That is more aligned that what we usually get from malloc.

Reply:
As per comment from Mike:
"32-byte alignment is required for avx2 operations.  On x86 at least, the library may be using avx2 even if the caller isn’t compiled with -mavx2.  We could instead change it so all the vector ops assume unaligned."